### PR TITLE
Editorial issues

### PR DESCRIPTION
**Pull request by gkellogg**
For #124 and #127.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/pull/130.html" title="Last updated on Feb 15, 2019, 1:25 AM UTC (7031cc2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/130/5776e7f...7031cc2.html" title="Last updated on Feb 15, 2019, 1:25 AM UTC (7031cc2)">Diff</a>

### CODE REVIEW

**Comment by gkellogg** [comment]
Thus far, this PR addresses comments in #124, although some changes are not made, as indicated in https://github.com/w3c/json-ld-syntax/issues/124#issuecomment-460362383 and https://github.com/w3c/json-ld-syntax/issues/124#issuecomment-463326891.

File: index.html [hunk 1]
```diff
@@ -1843,11 +1857,122 @@ <h1>Advanced Concepts</h1>
     </pre>
   </aside>
 
+  <p>The outer array is standard for a document in
+    <a href="expanded-document-form">expanded document form</a>
+    and <a href="flattened-document-form">flattened document form</a>,
+    and may be necessary when describing a disconnected graph,
+    where nodes may not reference each other. In such cases, using
+    a top-level dictionary with a @graph property can be useful for saving
+    the repetition of @graph. See <a href="#embedding" class="sectionRef"></a>
```
Comment (pchampin) [inline index.html]: `@graph` should be in a `code` tag...

File: index.html [hunk 2]
```diff
@@ -980,14 +980,22 @@ <h2>IRIs</h2>
...
+
   <p class="note"><a>Properties</a>, values of <code>@type</code>,
     and values of <a>properties</a> with a <a>term definition</a>
     that defines them as being relative to the <a>vocabulary mapping</a>,
     may have the form of a <a>relative IRI</a>, but are resolved using the
     <a>vocabulary mapping</a>, and not the <a>base IRI</a>.</p>
 
   <p>A <a>string</a> is interpreted as an <a>IRI</a> when it is the
-    value of an <a>dictionary member</a> with the key<code>@id</code>:</p>
+    value of an <a>dictionary member</a> with the key <code>@id</code>:</p>
```
Comment (pchampin) [inline index.html]: replace "an" with "a"

**Comment by iherman** [comment]
Looking at the new section before the (new) example 19, shouldn't it be "...with a `@graph` property can be useful for saving the repetition of `@context`"? (The current sentence repeats `@graph` on both places.)

**Comment by iherman** [comment]
See also https://github.com/w3c/json-ld-syntax/issues/127#issuecomment-463601229

**Comment by gkellogg** [comment]
Should be ready to go now.

**Pull request merged**